ldk/node/examples/clipboard/plugin.json

```diff
@@ -0,0 +1,13 @@
+{
+  "author": "Olive",
+  "created": "2020-06-22T00:00:00Z",
+  "dependencies": [],
+  "description": "This is an example controller written in Go.",
```

Comment:
Node.js example says "written in Go" and uses "controller" instead of "loop."  Also line 8 says "Example Go"

Reply:
Ah! Thanks, fixed!